samples/bot-proactive-messaging-teamsfx/bot/teamsBot.js

```diff
@@ -3,6 +3,11 @@
 
 const { TeamsActivityHandler, TurnContext } = require("botbuilder");
 
+const path = require('path');
+
+// Read botFilePath and botFileSecret from .env file.
+require('dotenv').config({ path: '.env.local' });
```

Comment:
You are explicitly specifying .env.local
I think, it will not work after code deployment.